L2SIVacuum/POUs/Functions/Valves/FB_VCN_VAT590.TcPOU

```diff
@@ -32,32 +49,48 @@ VAR
 	bRemAcsToggle : BOOL;              // Remote access control bit
 	//timer for zero function 
 	tZeroTogg : TON := (PT := T#1S);
-	//Rising edge trigger for Zero function
-	rtZero : R_TRIG;
 	//timer for remote function
 	tRemAcsTogg : TON := (PT := T#500MS);
 	//used for logging
 	ePreviousState : E_VCN_VAT590;
 	
+	(*CoE Interface*)
+	(*Link this to your VAT590 reference variables under InfoData.*)
+	i_stVat590PlcDriveRef AT %I*: ST_PlcDriveRef;
```

Comment:
Does the user need some additional guidance here in the comment e.g. which IO node's InfoData to link here? I remember the vacuum project terminals being labeled by location rather than by function. Maybe this would not be needed but I myself do not know which terminals are used to control VAT590.

Reply:
I added this two comments, I hope it will be more clear to the user now. 
	//Connect i_stVat590PlcDriveRef.aNetId to the VAT590 Valve InfoData^AdsAddr^netId
	//Connect i_stVat590PlcDriveRef.nSlaveAddr to the VAT590 Valve InfoData^AdsAddr^port